docs/guides/contributor/workflows.md

```diff
@@ -98,7 +98,7 @@ But at this point, you have certainly entered the "I know what I am doing" stage
 
 ## Run the `test-remote` suite against a remote cluster
 
-See [the `test-remote` suite of tests](./test-remote), a document dedicated to this test suite.
+See [the `test-remote` suite of tests](https://github.com/opstrace/opstrace/tree/main/test/test-remote), a document dedicated to this test suite.
```

Comment:
@jgehrcke thoughts on this? Would you rather link directly to the README.md?

Reply:
This is fine / better than linking right to the README. Thanks!